uber/models.py

```diff
@@ -545,7 +545,7 @@ def next_badge_num(self, badge_type, old_badge_num):
 
             # Adjusts the badge number based on badges in the session
             for attendee in [m for m in chain(self.new, self.dirty) if isinstance(m, Attendee)]:
```

Comment:
There was a unit test for this, so it was working at some point but must have broken awhile back without us noticing.